client/src/components/Feed/MessagesModal/MessagesContainer.js

```diff
@@ -247,34 +270,35 @@ export const PrivateMessageContainer = styled.div`
   ${(props) =>
     props.isFromUserCard
       ? `
-    margin-left: 7.4rem;
+    // margin-left: 7.4rem;
```

Comment:
this is a comment too and cause confusion

Reply:
I will clean it!